packages/earn-controller/package.json

```diff
@@ -51,10 +51,12 @@
     "@ethersproject/providers": "^5.7.0",
     "@metamask/base-controller": "^8.2.0",
     "@metamask/controller-utils": "^11.12.0",
+    "@metamask/keyring-api": "^20.1.0",
     "@metamask/stake-sdk": "^3.2.1",
     "reselect": "^5.1.1"
   },
   "devDependencies": {
+    "@metamask/account-tree-controller": "^0.12.1",
     "@metamask/accounts-controller": "^33.0.0",
```

Comment:
Nit: should we remove AccountsController from dev, peer dependencies and tsconfig ?

Reply:
Yes, I meant to! Thanks